### DISCUSSION TITLE
New line with right_format variable

### DISCUSSION THREAD

**Discussion by mortymacs** [starship/starship#4338] [Q&A] [answered]
Hi,

I want to add a newline for the prompt.
When I add `$line_break`, it adds a new line, but the right side variables will go to new line too, while I want them to be in the same line as the left side variables.
I did many changes but I couldn't achieve what I wanted.

My format is:

```toml
format = """
$sudo\
$username\
[](bg:#3E517A fg:#ED6A5A)\
$directory\
[](fg:#3E517A bg:#526CA3)\
$vcsh\
$git_branch\
$git_commit\
$git_state\
$git_metrics\
$git_status\
[](fg:#526CA3)$line_break """

right_format = """
$cmake\
$golang\
$helm\
$lua\
$python\
$red\
$rust\
$container\
$docker_context\
$kubernetes\
$terraform\
$vagrant\
$aws\
$gcloud\
$env_var\
$cmd_duration\
$jobs\
$status\
"""
```

![image](https://user-images.githubusercontent.com/1261448/188079703-f3a4181a-f742-4474-9455-dda2af59be98.png)

**Comment by davidkna** [comment, marked as answer]
Please use the [fill module](https://starship.rs/config/#fill) to right-align modules in non-final prompt lines.

**Comment by mortymacs** [comment]
Thanks!
It's worked!

```toml
format = """
$sudo\
$username\
[](bg:#3E517A fg:#ED6A5A)\
$directory\
[](fg:#3E517A bg:#526CA3)\
$vcsh\
$git_branch\
$git_commit\
$git_state\
$git_metrics\
$git_status\
[](fg:#526CA3) $fill\
$cmake\
$golang\
$helm\
$lua\
$python\
$red\
$rust\
$container\
$docker_context\
$kubernetes\
$terraform\
$vagrant\
$aws\
$gcloud\
$env_var\
$cmd_duration\
$jobs\
$status\
"""
```

![image](https://user-images.githubusercontent.com/1261448/188118770-978f8c08-d198-4a51-81c2-6e18ac266aa0.png)
